WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/ScanViewModelTest.kt

```diff
@@ -146,75 +155,72 @@ class ScanViewModelTest : BaseUnitTest() {
             }
 
     @Test
-    fun `given no network, when scan state fetched over empty scan state, then app reaches no connection state`() =
-            test {
-                whenever(scanStore.getScanStateForSite(site)).thenReturn(null)
-                whenever(fetchScanStateUseCase.fetchScanState(site)).thenReturn(flowOf(Failure.NetworkUnavailable))
-                val uiStates = init().uiStates
+    fun `given no network, when scan state fetched on init, then app reaches no connection state`() = test {
```

Comment:
Minor (🔍): You have used the `not invoked by user` vs `invoked by user` on your other tests, as such I believe this is a better alternative to `on init` and would suggest to replace the `fetched on init` with `fetch not invoked by user`. Also, the fact that the parameter is called `invokedByUser` goes hand-in-hand with the `(not) invoked by user` when naming.

PS: Same applies to the test below and those 2 extra tests that still use the `on init` further below.

Reply:
Updated in a968f7c